src/main/java/com/drinchev/projectlabel/resources/ui/ProjectLabelStatusBarWidget.java

```diff
@@ -166,22 +168,21 @@ public void paintComponent(final Graphics graphics) {
             Dimension size = getSize();
             final Dimension arcs = new Dimension(8, 8);
 
-            // image
-            bufferedImage = ImageUtil.createImage(size.width, size.height, BufferedImage.TYPE_INT_ARGB);
+            FontMetrics metrics = graphics.getFontMetrics(font);
+
+            bufferedImage = ImageUtil.createImage(size.width, size.height - VERTICAL_MARGIN, BufferedImage.TYPE_INT_ARGB);
```

Comment:
wouldn't that need to be 

```java
bufferedImage = ImageUtil.createImage(size.width, size.height - (2*VERTICAL_MARGIN), BufferedImage.TYPE_INT_ARGB);
```
to be consistent?

Reply:
Yeah I messed up with `UIUtil.drawImage(` where the VERTICAL_MARGIN should've been added.

Fixed, Thanks!